libmscore/mscore.cpp

```diff
@@ -352,6 +355,9 @@ QQmlEngine* MScore::qml()
             qmlRegisterUncreatableType<Element>("MuseScore", 1, 0,
                "Element", tr("you cannot create an element"));
 
+            //classed enumerations
+            qmlRegisterUncreatableType<MSQE_TextStyleType>("MuseScore", 1, 0, "TextStyleType", tr("You can't create an enum"));
```

Comment:
Translatable string that won't be translated into 2.0.3 because of the language freeze.
Only shows up in Plugin Creator console if someone tries to misuse the exposed enum as a type and write `var a = new TextStyleType;`